index.html

```diff
@@ -43,35 +43,38 @@ <h2>Let's start building your software&hellip;</h2>
-            - experts in infrastructure
-            - experts in user requirement elicitation
-            - experts in storage solutions
-            - experts in ux
-            - we put it together making it easy for developers to deploy
-            - reduces cost of introducing new developers, any developer can follow code to production
+            Our team has experts in every area needed to build and run software in your organization.
+            We have experts in application design, infrastructure and cloud, user requirements elicitation, databases, and user experience design.
+        </p>
+        <p>
+            We bring all of these expertises with us, when we work with you.
+            By employing the principles of DevOps, each of our developers is empowered to build, deploy and maintain software for you, meaning that we deliver quickly and with great stability.
         </p>
     </div>
     <div class="point-of-differentiation">
         <img src="/img/explore-icon.svg">
         <h1>Technical counselling</h1>
         <p>
-            - xp from many startups
-            - building software from your needs first/concrete biz situation (green field v existing system)
-            - technical decisions: cloud, processes (workflow), security, future proofing, stability (testing)
-            - advise your cto or act your cto
+            We have put together a team with a lot of experience from a broad set of software businesses.
+            We know the things that commonly go wrong when running a development department and building software&mdash;and we use that knowledge to help you avoid these mistakes.
+        </p>
+        <p>
+            We always start with your concrete business situation, and help you take the best steps forwards, whether you have an existing piece of software to elaborate on, or are building a new one.
+        </p>
+        <p>
+            Our counselling covers all technical decisions, including infrastructure, work processes, security, future-proofing of applications, and how to increase and ensure stability of both software and process.
         </p>
     </div>
     <div class="point-of-differentiation">
         <img src="/img/extension-icon.svg">
         <h1>Plug and play</h1>
         <p>
-            - we are a team that already knows how to work together
-            - we train our employees in getting up to speed in new systems, infrastructures and langauges quickly
-            - you rent some time from the team, and the team will ensure a good working culture
-            - you communicate with someone who helps you go from desired business impact to concrete steps
-            - get started quickly, get rid of/replace us quickly
+            When we say that we <em>hit the ground running</em>, we mean it: we are a team that has established a strong culture of working together, and we bring that with us, when you hire us for work in your business.
```

Comment:
"strong culture of working together" --> "strong culture of teamwork and knowledge sharing"

Reply:
I like "working together", because it is in every sense... Not just that we have teamwork and knowledge sharing, but that we work well together in every sense (many more than these two).

I will change it to "strong culture of working well together"